src/application/variables/assignVarFromValue.ts

```diff
@@ -44,7 +44,20 @@ export const assignVarFromValue = (
   }
 
   // Set variable value
-  const varValue = typeof varSetting.value === 'string' ? `"${varSetting.value}"` : varSetting.value
+  const generatedValue = parseTpl({
+    template: varSetting.value,
+    oaOperation: oaOperation,
+    options: {
+      casing: globals?.variableCasing
+    }
+  })
+  const varValue =
+    varSetting?.value && hasTpl(varSetting.value)
+      ? generatedValue
+      : typeof varSetting.value === 'string'
+      ? `"${varSetting.value}"`
+      : varSetting.value
+  // const varValue = typeof varSetting.value === 'string' ? `"${varSetting.value}"` : varSetting.value
```

Comment:
remove if not needed anymore...

Reply:
Cleaned-up and discovered that the result was not a "string", which was expected, which is now also corrected.